admin/tests/acceptance/authenticated/users/get_test.js

```diff
@@ -56,10 +68,11 @@ module('Acceptance | authenticated/users/get', function (hooks) {
     const userNavigation = within(screen.getByLabelText("Navigation de la section détails d'un utilisateur"));
     assert.dom(userNavigation.getByRole('link', { name: 'Détails' })).exists();
     assert.dom(userNavigation.getByRole('link', { name: 'Profil' })).exists();
-    assert.dom(userNavigation.getByRole('link', { name: 'Participations' })).exists();
+    assert.dom(userNavigation.getByRole('link', { name: `Participations (1)` })).exists();
```

Comment:
```suggestion
    const expectedParticipationsNumber = 1
    assert.dom(userNavigation.getByRole('link', { name: `Participations (${expectedParticipationsNumber})` })).exists();
```

Le `1` seul est une valeur magique, l'on préfère une constate pour expliciter son sens